microlith/Dockerfile

```diff
@@ -163,11 +162,13 @@ WORKDIR /
 
 # LOKI:
 # From https://github.com/grafana/loki/blob/main/cmd/loki/Dockerfile
+# Note: we removed `/etc/loki` from the last line - we'll chown it later.
```

Comment:
is "chown" a misspell?

Reply:
No, it's a command we run later (https://github.com/couchbaselabs/observability/blob/b63dc43d7f1a1e23683dcd5af9c9a1f1b66b5fc5/microlith/Dockerfile#L335-L348)